.github/workflows/CITest.yml

```diff
@@ -81,37 +80,32 @@ jobs:
 
     - name: make
       if: startsWith(matrix.config.build-system, 'make')
-      shell: 'script -q -e -c "bash {0}"'
       run: |
-        ./make.sh;
-        make check;
+        ./make.sh
+        make check
+        sudo make install
```

Comment:
~~If it makes not too much trouble please use cmake here. The `Makefile` build is deprecated.~~ Ouh, misread what workflow this is :D Ignore it